### ISSUE TITLE
Getting "R Tools require .NET Core Runtime. Would you like to install it now?"

### ISSUE THREAD

**Issue by VLucet** [MikhailArkhipov/vscode-r#57]
Hi, I'm on Linux (pop os 20.10), I have insatted the extension and was prompted to install .NET. I have the folowing dotnet install:
```
➜  ~ dotnet --info                               

Host (useful for support):
  Version: 5.0.4
  Commit:  f27d337295

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.NETCore.App 5.0.4 [/snap/dotnet-runtime-50/11/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```
I have also added `export DOTNET_ROOT=/snap/dotnet-runtime-50/current` to my `.zshrc`
It seems that the extension is not finding the .NET runtime, would appreciate any help!

**Comment by MikhailArkhipov** [comment]
So what happens exactly? Any output in `R Tools Startup` or `R Tools` tabs?

**Comment by VLucet** [comment]
Sorry i wasn't clear, what happens is that I get the prompt to install .Net and that is all, even after reboot. I also tried disabling the checks for .Net in the settings but I get notified that the extension failed 5 times and that I will not try to start again.

**Comment by MikhailArkhipov** [comment]
Try this

- Open extensions folder (VS Code command)
- go to the extension folder, then `ls`. I believe it is `~/.vscode/extensions/mikhail-arkhipov.r-0.0.20/ls`
- try running `dotnet Microsoft.R.LanguageServer.dll`

does it start?

**Comment by VLucet** [comment]
Thanks, here is what I got when I do this:
```
Cannot use file stream for [/home/vlucet/.vscode/extensions/mikhail-arkhipov.r-0.0.20/ls/Microsoft.R.LanguageServer.runtimeconfig.json]: Permission denied
Invalid runtimeconfig.json [/home/vlucet/.vscode/extensions/mikhail-arkhipov.r-0.0.20/ls/Microsoft.R.LanguageServer.runtimeconfig.json] [/home/vlucet/.vscode/extensions/mikhail-arkhipov.r-0.0.20/ls/Microsoft.R.LanguageServer.runtimeconfig.dev.json]
```

**Comment by MikhailArkhipov** [comment]
It looks like you somehow don't have permissions to read `Microsoft.R.LanguageServer.runtimeconfig.json`? Perhaps reinstall extension (with deleting its folder after uninstall + restart vs code).

**Comment by VLucet** [comment]
So I unisntalled the app, deleted the folder, rebooted, then I reinstalled and still have the issue. I can open the json file in vscode:
```
{
  "runtimeOptions": {
    "tfm": "net5.0",
    "framework": {
      "name": "Microsoft.NETCore.App",
      "version": "5.0.0"
    }
  }
}
```

**Comment by AlFontal** [comment]
I am seeing the exact same issue, working on Ubuntu 20.04 on VS Code insiders. Did you find any solution @VLucet?

**Comment by VLucet** [comment]
No, I haven't

**Comment by trybik** [comment]
Same issue on macOS 11.3.1 with Homebrew-installed dotnet 5.0.203 (with `DOTNET_ROOT` set); however, here running the `dotnet Microsoft.R.LanguageServer.dll` in extension's `ls` subfolder seems to work fine (no output).

**Comment by MikhailArkhipov** [comment]
What happens if you set `r.dependencyChecks` to `false`? Does extension work?

**Comment by trybik** [comment]
Yes, seems like it does work w/  `r.dependencyChecks` set to `false` - nice, will try it out.

**Comment by benlansdell** [comment]
I have the exact same 'permissions' issue. I can read the json in a text editor though. Setting r.dependencyChecks to false doesn't appear to help for me.

Any other suggestions?
